tests/integration/ceph_base_file_test.go

```diff
@@ -259,11 +266,130 @@ func runFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.S
 	// Start the NFS daemons
 	testNFSDaemons(helper, k8sh, s, settings, filesystemName)
 
-	// Cleanup the filesystem and its clients
-	cleanupFilesystemConsumer(helper, k8sh, s, settings.Namespace, filePodName)
-	assert.NoError(s.T(), err)
-	downscaleMetadataServers(helper, k8sh, s, settings.Namespace, filesystemName)
-	cleanupFilesystem(helper, k8sh, s, settings.Namespace, filesystemName)
+	t := s.T()
+	ctx := context.TODO()
+
+	// TODO: there is a regression here where MDSes don't actually scale down, and this test
+	// wasn't catching it. Enabling this test causes the controller to enter into a new reconcile
+	// loop and makes the next phase of the test take much longer than it should, making it flaky.
+	// Rook issue https://github.com/rook/rook/issues/9857 is tracking this issue.
+	// t.Run("filesystem should be able to be scaled down", func(t *testing.T) {
+	// 	downscaleMetadataServers(helper, k8sh, t, settings.Namespace, filesystemName)
+	// })
```

Comment:
@travisn the commented-out `downscaleMetadataServers` isn't intended to be removed but fixed later. It's related to this code.